terraform/modules/load_balancer/README.md

```diff
@@ -0,0 +1,34 @@
+## Requirements
+
+No requirements.
+
+## Providers
+
+| Name | Version |
+|------|---------|
+| <a name="provider_helm"></a> [helm](#provider\_helm) | n/a |
+| <a name="provider_kubernetes"></a> [kubernetes](#provider\_kubernetes) | n/a |
+
+## Modules
+
+No modules.
+
+## Resources
+
+| Name | Type |
+|------|------|
+| [helm_release.load-balancer](https://registry.terraform.io/providers/hashicorp/helm/latest/docs/resources/release) | resource |
+| [kubernetes_config_map.tcp](https://registry.terraform.io/providers/hashicorp/kubernetes/latest/docs/resources/config_map) | resource |
+
+## Inputs
+
+| Name | Description | Type | Default | Required |
+|------|-------------|------|---------|:--------:|
+| <a name="input_enable_mqtt_adapter"></a> [enable\_mqtt\_adapter](#input\_enable\_mqtt\_adapter) | Used to enable the MQTT adapter. | `bool` | n/a | yes |
+| <a name="input_enhanced_mqtt_load_balancer"></a> [enhanced\_mqtt\_load\_balancer](#input\_enhanced\_mqtt\_load\_balancer) | Configuration options for the enhanced MQTT load balancer.<br>  chart\_version: Version of the chart to deploy.<br>  replicaCount: Number of replicas to deploy.<br>  port\_configs: List of MQTT port config objects for the enhanced MQTT load balancer service.<br>  tcp\_configmap\_data: Data of the TCP configMap for the enhanced MQTT load balancer. | <pre>object({<br>    chart_version = string<br>    replicaCount = number<br>    port_configs = list(object({<br>      name       = string,<br>      port       = number,<br>      targetPort = number<br>    }))<br>    tcp_configmap_data = map(string)<br>  })</pre> | n/a | yes |
```

Comment:
can we document the specific settings of the load balancer in an additional table and reference the lb object here with a minimum description?

Reply:
I would prefer not to change anything here, as this readme is auto generated